PRIVACY.txt

```diff
@@ -37,7 +37,7 @@ This SERVICE makes use of third-party data. All data used in the
 creation of this SERVICE comes from sources that allow its full use
 in the manner done so by the SERVICE. Specifically, the data for this
 SERVICE comes from Wikidata and Wikipedia. Wikidata states that, "All
-structured data in the main, property and lexeme namespaces is made
+structured data in the main, property and lexeme namespaces are made
```

Comment:
Checking this now :) This is a direct quote, so it should remain as is. As data is "uncountable" even though obviously we should count it, I think that "is" is also correct here :)

Reply:
Got it! I'll keep the quote as is. Thanks for the feedback.